packages/components/src/index.tsx

```diff
@@ -0,0 +1,73 @@
+export * from "./AnimatedPresence";
```

Comment:
This file is required for ESM support. When creating new components in the future you will have to update this file, along with the package.json file to correctly export your component. I will update the documentation to reflect this change.